modules/aleph/src/Aleph/AlephDebtMaterial.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drupal\aleph\Aleph;
+
+/**
+ * Class AlephDebtMaterial.
+ *
+ * Material information related to debt.
```

Comment:
Mention that this class exists because we only have partial material information.

Also, didn't the response include more properties than the title of the material? How about some sort of ID?